public_html/wp-content/plugins/pattern-directory/includes/notifications.php

```diff
@@ -0,0 +1,243 @@
+			}
+		}
+		$reasons = array_map(
+			function( \WP_Term $reason ) {
+				return wp_strip_all_tags( $reason->description );
+			},
+			$reasons
+		);
+		$reasons = array_unique( $reasons );
+		$reason = trim( implode( "\n", $reasons ) );
+	} else {
+		// If it doesn't have flags, it must have gotten here by getting marked as spam.
+		$spam_term = get_term_by( 'slug', '4-spam' );
+		$reason = wp_strip_all_tags( $spam_term->description );
+	}
+
+	if ( ! $reason ) {
+		$reason = get_default_reason_description();
+	}
+
+	$subject = esc_html__( 'Pattern being reviewed', 'wporg-patterns' );
+
+	$message = sprintf(
+		// translators: Plaintext email message. Note the line breaks. 1. Pattern title; 2. Pattern URL;
+		esc_html__( 'Hi there!
+
+Thanks for submitting your pattern. Unfortunately, your pattern, %1$s, has been flagged for review due to the following reason(s):
+
+%2$s
+
+Your pattern has been unpublished from the Block Pattern Directory at this time, and will receive further review. If the pattern meets the guidelines, we will re-publish it to the Block Pattern Directory. Thanks for your patience with us volunteer reviewers!', 'wporg-patterns' ),
+		esc_html( html_entity_decode( $pattern_title ) ),
+		esc_html( html_entity_decode( $reason ) )
+	);
+
+	if ( $locale ) {
+		restore_current_locale();
+	}
+
+	send_email( $email, $subject, $message );
+}
+
+/**
+ * Notify when a pattern has been unlisted.
+ *
+ * @param \WP_Post $post
+ *
+ * @return void
+ */
+function notify_pattern_unlisted( $post ) {
+	$author = get_user_by( 'id', $post->post_author );
+	if ( ! $author ) {
+		return;
+	}
+
+	$email  = $author->user_email;
+	$locale = get_user_locale( $author );
+
+	$pattern_title = get_the_title( $post );
+
+	$reasons = get_the_terms( $post, REASON );
+	$reason = '';
+	if ( ! empty( $reasons ) ) {
+		$reason_term = reset( $reasons );
+		$reason = wp_strip_all_tags( $reason_term->description );
+	}
+
+	if ( ! $reason ) {
+		$reason = get_default_reason_description();
+	}
+
+	if ( $locale ) {
+		switch_to_locale( $locale );
+	}
+
+	$subject = esc_html__( 'Pattern unlisted', 'wporg-patterns' );
+
+	$message = sprintf(
+		// translators: Plaintext email message. Note the line breaks. 1. Pattern title; 2. Pattern URL;
+		esc_html__( 'Hello,
+
+Your pattern, %1$s, has been unlisted from the Block Pattern Directory due to the following reason:
+
+%2$s
+
+If you would like to resubmit your pattern, please make sure it follows the guidelines:
+
+%3$s', 'wporg-patterns' ),
+		esc_html( html_entity_decode( $pattern_title ) ),
+		esc_html( html_entity_decode( $reason ) ),
+		'https://wordpress.org/patterns/about/'
+	);
+
+	if ( $locale ) {
+		restore_current_locale();
+	}
+
+	send_email( $email, $subject, $message );
+}
+
+/**
+ * Wrapper for wp_mail.
+ *
+ * @param string $to
+ * @param string $subject
+ * @param string $message
+ *
+ * @return void
+ */
+function send_email( $to, $subject, $message ) {
+	$message = html_entity_decode( $message );
```

Comment:
Something is still encoding the apostrophe in the message— the email I got says:

`This pattern doesn&#039;t meet the guidelines for the pattern directory.`

Reply:
To decode quotes you have to set the `ENT_QUOTES` flag.

```suggestion
	$message = html_entity_decode( $message, ENT_QUOTES );
```

Reply:
Today I learned. Thanks @ocean90!